docs/source/usage/workflows/memoryPerDevice.rst

```diff
@@ -0,0 +1,115 @@
+.. _usage-workflows-memoryPerDevice:
+
+Calculating the memory requirement per device
+---------------------------------------------
+
+.. sectionauthor:: Marco Garten
+
+The planning of simulations for realistically sized problems requires a careful estimation of memory usage and is often a trade-off between resolution of the plasma, overall box size and the available resources.
+The file :ref:`memory_calculator.py <usage-python-utils>` contains a class for this purpose.
+
+The following paragraph shows the use of the ``MemoryCalculator`` for the ``4.cfg`` setup of the :ref:`FoilLCT example <usage-examples-foilLCT>` example.
+
+.. code:: python
+
+    from picongpu.utils import MemoryCalculator
```

Comment:
you know that you can check in this file as a script, `literalinclude` it here and even *run it at sphinx build time* to generate the *output below*? :)